.github/workflows/flagd-check.yml

```
@@ -0,0 +1,43 @@
+name: Flagd Checks
+
+on:
+  push:
+    paths:
+      - 'crates/flagd/**'
+      - '.github/workflows/flagd-check.yml'
+  pull_request:
+    paths:
+      - 'crates/flagd/**'
+      - '.github/workflows/flagd-check.yml'
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Update git submodules
+        run: git submodule update --init --recursive
```

```suggestion
      - uses: actions/checkout@v4
        with:
          submodules: recursive
```

I believe this should also work.

https://github.com/actions/checkout?tab=readme-ov-file#usage